float32 x
float32 y
float32 theta
float32 v
float32 w
---
float64[] x_points
float64[] y_points